ui/src/main/java/com/wireguard/android/fragment/AppListDialogFragment.kt

```diff
@@ -43,6 +44,10 @@ class AppListDialogFragment : DialogFragment() {
                     val launcherIntent = Intent(Intent.ACTION_MAIN, null)
                     launcherIntent.addCategory(Intent.CATEGORY_LAUNCHER)
                     val resolveInfos = pm.queryIntentActivities(launcherIntent, 0)
+                    if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.M) {
```

Comment:
Add an import for `android.os.Build` please.

Reply:
Done